extensions/pgx_pgmq/build-extension.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
similar to this script

https://github.com/postgresml/postgresml/blob/master/pgml-extension/build_extension.sh

Reply:
It uses a container to build the extension, and outputs to the local filesystem the `.deb` package